playbooks/templates/.env.j2

```
@@ -176,19 +176,20 @@ SKYNET_ACCOUNTS_LOG_LEVEL={{ webportal_common_config.accounts_log_level }}
 # - 'false': All users incl. anonymous users can access the portal (access to portal is public)
 # REQUIRED: manually defined by user, or default is set in docker-compose file
 ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access | string | lower }}
+{% endif %}
 
 {% if webportal_common_config.accounts_limit_access | string | lower != 'false' %}
```

```suggestion
{% if webportal_common_config.accounts_limit_access | deafult('') | string | lower != 'false' %}
```

I think we need a default here if it is undefined in LastPass.